.clang-tidy

```diff
@@ -1,5 +1,5 @@
 ---
-Checks:         '*,-fuchsia-overloaded-operator,-fuchsia-default-arguments,-llvm-include-order,-llvm-header-guard,-readability-redundant-declaration,-misc-unused-parameters,-modernize-use-override,-cppcoreguidelines-pro-type-vararg,-google-default-arguments,-google-runtime-references,-google-readability-todo,-google-readability-namespace-comments,-clang-analyzer-alpha.core.CastToStruct,-clang-diagnostic-pragma-once-outside-header,-google-build-using-namespace,-cppcoreguidelines-pro-type-static-cast-downcast,-cppcoreguidelines-pro-bounds-pointer-arithmetic,-cert-err58-cpp,-cppcoreguidelines-pro-bounds-constant-array-index,-cppcoreguidelines-pro-type-reinterpret-cast,-misc-misplaced-widening-cast,-clang-analyzer-alpha.deadcode.UnreachableCode'
+Checks:         '*,-android-cloexec-fopen,-fuchsia-overloaded-operator,-fuchsia-default-arguments,-llvm-include-order,-llvm-header-guard,-readability-redundant-declaration,-misc-unused-parameters,-modernize-use-override,-cppcoreguidelines-pro-type-vararg,-google-default-arguments,-google-runtime-references,-google-readability-todo,-google-readability-namespace-comments,-clang-analyzer-alpha.core.CastToStruct,-clang-diagnostic-pragma-once-outside-header,-google-build-using-namespace,-cppcoreguidelines-pro-type-static-cast-downcast,-cppcoreguidelines-pro-bounds-pointer-arithmetic,-cert-err58-cpp,-cppcoreguidelines-pro-bounds-constant-array-index,-cppcoreguidelines-pro-type-reinterpret-cast,-misc-misplaced-widening-cast,-clang-analyzer-alpha.deadcode.UnreachableCode'
```

Comment:
https://clang.llvm.org/extra/clang-tidy/checks/android-cloexec-fopen.html

Not very applicable (we never exec) and probably doesn't work on windows